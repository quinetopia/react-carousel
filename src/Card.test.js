import React from "react";
import { render } from "@testing-library/react";
import Card from "./Card";
import image1 from "./image1.jpg";

const card = {caption: "Test caption", src : image1, cardIdx: 1}
const total = 1;

it("renders without crashing", function() {

  render(<Card 
            caption={card.caption}
            src={card.src}
            currNum={cardIdx + 1}
            totalNum={total}/>);
});

it("matches snapshot", function() {
  const { asFragment } = render(<Card 
                                  caption={card.caption}
                                  src={card.src}
                                  currNum={cardIdx + 1}
                                  totalNum={total}/>);
  expect(asFragment()).toMatchSnapshot();
});