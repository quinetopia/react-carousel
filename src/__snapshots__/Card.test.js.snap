// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Test caption
    </h4>
    <img
      alt="Test caption"
      class="Card-image"
      src="image1.jpg"
    />
    <small
      class="Card-small"
    >
      Image 2 of 1.
    </small>
  </div>
</DocumentFragment>
`;
